assets/web/_bevy_dev/auto_reload.js

```diff
@@ -0,0 +1,72 @@
+// Automatically reload when the server restarts
+// Adopted from <https://github.com/trunk-rs/trunk/pull/231>
+(function () {
+  const baseUrl = window.location.host + '/_bevy_dev/websocket';
+  const websocketUrl = 'ws://' + baseUrl;
```

Comment:
```suggestion
  const baseUrl = window.location.host + "/_bevy_dev/websocket";
  const websocketUrl = "ws://" + baseUrl;
```

The rest of the code uses double quotes, so I would like to remove the single quotes here for consistency.